src/jsMain/kotlin/io/beatmaps/shared/review/main.kt

```diff
@@ -94,8 +95,10 @@ class ReviewTable : RComponent<ReviewTableProps, ReviewTableState>() {
                 attrs.container = resultsTable
                 attrs.renderElement = InfiniteScrollElementRenderer { rv ->
                     reviewItem {
-                        obj = rv
-                        userId = rv?.creator?.id ?: -1
+                        obj = rv?.apply {
```

Comment:
I think this should probably use copy rather than apply to keep the property a val